functions/expiredMediaChecker/index.js

```diff
@@ -0,0 +1,50 @@
+const { updateTranscription } = require("../sttChecker");
+
+const expiredMediaChecker = (bucket, admin) => {
+  bucket.getFiles().then((data) => {
+    const files = data[0];
+
+    files.forEach((file) => {
+      const dateCreated = new Date(file.metadata.timeCreated);
+
+      let expiryDate = new Date(dateCreated);
+      expiryDate.setDate(dateCreated.getDate() + 60);
+
+      const isDeletableContentType = (contentType) => {
```

Comment:
Think this can be a little simpler: 
`const isDeletableContentType = (contentType) => contentType ? contentType.includes("audio") || contentType.includes("video") : false`

and perhaps could be moved outside of the scope of this function?

Reply:
great shout, thanks!